src/main/reactjs/src/components/i18n/LangSelector.tsx

```diff
@@ -34,7 +34,7 @@ export const LangSelector: FC = () => {
         value={getCurrentLang()}
         onChange={handleLangChange}
         className="lang-selector__select"
-        data-testid="lang-selector"
+        dataTestId="lang-selector"
```

Comment:
data-testid vai dataTestId?

Reply:
Muutin `dataTestId`:ksi, koska `Dropdown.tsx` olettaa saavansa sellaisen propin. Aiempi `LangSelector`in cypress-testi failasi tähän.

Reply:
Ja tuo `dataTestId` siis tietysti löytyy `DropdownProps`-interfacesta meidän koodista.
Ehkä oikea ratkaisu voisi olla muuttaa koodia niin että tarjotaan `DropdownProps` alta vain meidän "omia" attribuutteja ja loput attribuutit saadaan extendaamalla MUIn omasta `SelectProps` interfacesta.

Reply:
Joo, olisi hyvä tehdä tämä muutos.

Reply:
Jep, muokkaan tähän samaan PR:ään.